src/app/scripts/index.js

```diff
@@ -18,7 +18,123 @@ angular.module('whatsPup', ['ngAnimate', 'ngCookies', 'ngTouch', 'ngSanitize', '
         templateUrl: 'app/view/addclient.html',
         controller: 'AddClientCtrl',
         controllerAs: 'client'
-    });
+    })
 
+    .state('navbar', {
+        url: '/nav',
+        templateUrl: 'app/view/navbar.html',
+        controller: 'NavCtrl',
+        controllerAs: 'nav'
+    })
+
+    .state('owner', {
+    url: '/owner',
+    templateUrl: 'app/view/ownerpage.html',
+    controller: 'OwnerBtnCtrl',
+    controllerAs: 'ownerbtn'
+})
     $urlRouterProvider.otherwise('/');
+})
+
+.factory('Auth', function ($firebaseObject, $state) {
```

Comment:
Would be a lot easier to read if this `Auth` service was defined in a separate file...